### PR TITLE
Update remaining `master` references to `primary`

### PR DESCRIPTION
**Pull request by PingXie**

### CODE REVIEW

**Review by madolson — APPROVED** [review]
lgtm

File: src/valkey-cli.c [hunk 1]
```diff
@@ -4993,7 +4991,7 @@ clusterManagerMoveSlot(clusterManagerNode *source, clusterManagerNode *target, i
          * unblocked with the role change error. */
         success = clusterManagerSetSlot(source, target, slot, "node", err);
         if (!success && err) {
-            const char *acceptable[] = {"ERR Please use SETSLOT only with masters.", "UNBLOCKED"};
+            const char *acceptable[] = {"ERR Please use SETSLOT only with ", "UNBLOCKED"};
```
Comment (enjoy-binbin) [inline src/valkey-cli.c]: missing this?
```suggestion
            const char *acceptable[] = {"ERR Please use SETSLOT only with primaries.", "UNBLOCKED"};
```

**Review by enjoy-binbin — APPROVED** [review]
LGTM.

File: src/valkey-cli.c (hunk 1 above)
Comment (PingXie) [reply src/valkey-cli.c]: Actually I can't change the error string as that might break some clients. I will add both strings (primaries and masters) next.

**Pull request merged**